prov/gni/test/rdm_sr.c

```diff
@@ -111,9 +114,23 @@ void rdm_sr_setup_common_eps(void)
 	target = malloc(BUF_SZ * 3); /* 3x BUF_SZ for multi recv testing */
 	assert(target);
 
+	dest_iov = malloc(sizeof(struct iovec) * IOV_CNT);
+	assert(dest_iov);
```

Comment:
I see a lot of assertions used for checking NULL pointers.  Since these are run time errors instead of program errors, actual return value checking is preferred over assert because "assert()" does nothing if NDEBUG is defined.